doc/viewer.rst

```diff
@@ -241,14 +241,19 @@ Camera Positioning/Orientation
   :param obj: Must be an object implementing a *center* method returning the center of the object, e.g. an instance of :class:`pv.mol.MolView`, :class:`pv.mol.Mol`
 
   
-.. function:: pv.Viewer.autoZoom()
+.. function:: pv.Viewer.autoZoom([ms])
 
   Adjusts the zoom level such that all objects are visible on screen and occupy as much space as possible. The center and orientation of the camera are not modified.  
+  
+  :param ms: if provided and non-zero defines the time in which the camera zoom level moves from thecurrent zoom level to the target zoom. If zero, the zoom is immediately set to the target zoom. If no value is provided it will go off of the animateTime option passed into the viewer.
```

Comment:
Thanks for also modifying the docs! I'd rephrase this slightly to something like this:


```:param ms: if provided and non-zero defines the time in which the camera zoom level moves from the current zoom level to the target zoom. If zero, the zoom is immediately set to the target zoom. If no value is provided it will use the default animation time of the viewer.```